media/js/src/levelsdashboard.tsx

```diff
@@ -43,53 +44,89 @@ export const LevelsDashboard: React.FC = () => {
 
     return (
         <>
-            <div className="d-flex flex-column min-vh-100 justify-content-center
-                            align-items-center" data-testid={'LevelsDashboard'}>
-                <div className="card" style={{width: '20rem', height: '10rem'}}>
-                    <div className="card-body">
-                        <div className="text-center">
-                            <p className="card-text">LEVEL I:</p>
-                            <p>NOVICE</p>
-                            <span className="float-end">
-                                {completedNovice} / {noviceCount}
-                            </span>
-                            <div><a className={'btn btn-primary'}
-                                href={'/questions/0'}>Level I</a>
+            <HomeBanner />
+            <section className="container content-body" id="maincontent">
```

Comment:
Same thing here. Just add `data-testid={'LevelsDashboard'}` to the section tag and we're good to go!

Reply:
Fixed